src/microbe_stage/Patch.cs

```diff
@@ -648,12 +652,14 @@ public class PatchSnapshot : ICloneable
     public Dictionary<Species, SpeciesInfo> RecordedSpeciesInfo = new();
 
     public BiomeConditions Biome;
+    public string Background;
```

Comment:
How is old save compatibility going to be handled here? Maybe this could be nullable and the Patch.Background property could fallback to the old biome template read in that situation?